dynamo-migration-lambda/app/RdsMigration.py

```diff
@@ -0,0 +1,154 @@
+# TODO: PRMT-4648 - THIS IS A ONE-TIME MIGRATION SCRIPT LAMBDA. DELETE THIS .PY FILE AFTER USAGE!
+import os
+
+import boto3
+import logging
+import datetime
+import psycopg2
+
+from enum import Enum
+from dateutil import parser
+from zoneinfo import ZoneInfo
+from dataclasses import dataclass
+
+AWS_REGION = 'eu-west-2'
+TARGET_ENVIRONMENT = os.environ["ENVIRONMENT_NAME"]
+NEW_TABLE_NAME = f'{TARGET_ENVIRONMENT}-ehr-transfer-tracker'
+
+ssm_client = boto3.client("ssm")
+dynamo_client = boto3.client("dynamodb", region_name=AWS_REGION)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+conversations_to_delete = {}
+
+class MessageRowItem(Enum):
+    INBOUND_CONVERSATION_ID = 0
+    LAYER = 1
+    INBOUND_MESSAGE_ID = 2
+    CREATED_AT = 3
+    UPDATED_AT = 4
+    DELETED_AT = 5
+    PARENT_ID = 6
+
+
+@dataclass
+class DatabaseCredentials:
+    user: str
+    password: str
+    host: str
+    database: str
+
+
+def _get_new_datetime(old_datetime: datetime) -> str:
```

Comment:
```suggestion
def _get_new_datetime(old_datetime: datetime) -> Optional[str]:
```
when old_datetime is None this function will return None implicitly, so probably better to make it clear so that IDE could help us do type check

Reply:
Added Optional[str] as suggested.